packages/koa-metrics/src/index.ts

```diff
@@ -0,0 +1,58 @@
+import {StatsD} from 'hot-shots';
+import {Context} from 'koa';
+
+import {tagsForRequest, tagsForResponse} from './tags';
+import {initTimer, Timer} from './timing';
+import {instrumentContentLength} from './content';
+
+export {Tags} from './tags';
+
+export interface Options {
+  prefix: string;
+  host: string;
+  skipInstrumentation?: boolean;
+}
+
+export default function metrics({
```

Comment:
should we maybe take a basic `logger` interface (defaulting to console.log) and also output the stats there?

Reply:
I think we should do that for development, yeah.